pkg/pulumiyaml/sort.go

```diff
@@ -256,3 +248,11 @@ func isConfigNodeEnv(n graphNode) bool {
 	_, ok := n.(configNodeEnv)
 	return ok
 }
+
+func stripConfigNamespace(s string) string {
```

Comment:
Does this mean that `aws:region` and `my-proj:region` are actually the same node? This seems scary to me.

Reply:
True, though we can't get the project name due to the lack of context. We could store the first prefix to hit this as in a var, and then if the same stripped key, check its prefix against the var and error if different?

Reply:
I think we should have a way to have both `aws:region` and `my-proj:region`. We **need** to be able to express `aws:region` and `aws-native:region` at the same time.

Reply:
Actually this path is specifically for `pulumi up` I believe, so we should have access to the context